routes/course-column-options.js

```diff
@@ -25,9 +11,14 @@ client.connect(err => {
 */
 router.get('/', async (req, res) => {
   try {
-    const sql = 'SELECT * FROM course ORDER BY start_year DESC';
-    let courses = await client.query(sql);
-    courses = courses.rows;
+    const sql = 'SELECT * FROM courses ORDER BY start_year DESC';
+    let courses = await sequelize
+      .query(sql, {
+        type: sequelize.QueryTypes.SELECT
+      })
+      .then(courses => {
```

Comment:
This `.then` block is not needed. `.query()` should return a promise and `(courses)` (argument) is what will be returned to `let courses` via `await`